### DISCUSSION TITLE
Display relative international dates

### DISCUSSION THREAD

**Discussion by hchauvat** [quasarframework/quasar#14949] [Ideas / Proposals]
Hi, 

Currently working with Quasar quite a bit, I haven't found an easy way to render dates in a relative and international manner.

My idea is to have a method that would take a Date or a string and a language in parameters and return a string such as "x seconds/minutes/hours/days/months/years ago" in the correct language, based on the `Int.RelativeTimeFormat` API (see [doc](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat)).

I know there already is the `getDateDiff` method, but I feel like it could do more.

I already have a fully working piece of code. I could improve it, add some tests and make it Quasar-ready.

Let me know if that's something you think could be of interest for the project.

**Comment by zanzara** [comment]
Do you know this for international formatting?
https://vue-i18n.intlify.dev/guide/essentials/datetime.html
It works very well
